package/kernel/linux/modules/usb.mk

```diff
@@ -73,10 +73,10 @@ $(eval $(call KernelPackage,usb-phy-nop))
 define KernelPackage/usb-phy-qcom-dwc3
```

Comment:
I'd personally already rename the kmod here as well, in sync with the kconfig/ko file, and only move it to ipq806x target in that separate patch.